src/services/note/query.ts

```diff
@@ -0,0 +1,12 @@
+// src/services/note/query.ts
+import { queryOptions } from "@tanstack/react-query";
+
+import { NoteResponse } from "@/types/note";
+
+import { getNote } from ".";
+
+export const getNoteOptions = (noteId: number) =>
+  queryOptions<NoteResponse>({
+    queryKey: ["noteDetail", noteId],
+    queryFn: () => getNote(noteId),
+  });
```

Comment:
쿼리 키 만든 걸로 수정해주시면 좋을 것 같아요. 이렇게 직접 쓰이는 부분이 있으면 중복으로 관리해야 하기 때문에 쿼리 키 관리를 한 곳에서 하는 의미가 없어질 수 있을 것 같아요.

```suggestion
export const getNoteOptions = (noteId: number) =>
  queryOptions<NoteResponse>({
    queryKey: noteKeys.detail(noteId),
    queryFn: () => getNote(noteId),
  });
```